shark/examples/shark_inference/stable_diffusion/main.py

```diff
@@ -52,22 +52,23 @@ def get_models():
                 model_name=UNET_FP32
             )
         else:
+            iree_extra_args = [
+                "--iree-flow-enable-conv-nchw-to-nhwc-transform",
+                "--iree-flow-enable-padding-linalg-ops",
+                "--iree-flow-linalg-ops-padding-size=16",
+            ]
+            if len(args.iree_vulkan_target_triple) > 0:
```

Comment:
This can be passed globally—rather than passed individually to the models. Create a global list - IREE_EXTRA_ARGS and add the target triple there. Rest args can be appended.